src/components/Loader.tsx

```diff
@@ -19,21 +21,24 @@ const rippleLoaderChild = {
 /**
  * Loading component.
  */
-const Loader = ({ size = 32, style }: LoaderProps) => {
+const Loader = ({ size = 32, style, cssRaw }: LoaderProps) => {
```

Comment:
The `style` prop is now unused in the only place that calls `<Loader>`. Can we remove it?

Additionally, that passed style (`marginRight: '15px'`) used to apply to the outer div, but has been moved to the inner div. I tested the difference locally, and it looks like `marginRight` doesn't have any effect on the inner div, but it did on the outer div.

Reply:
Thanks for catching this! @raineorshine, there's been a good number of merge requests since the Loader commit, is it okay if I don't squash the fix into the original commit?

Reply:
Sure, that's fine!

FYI For the future I do prefer rebasing on `main` instead of merge commits unless a branch has fallen really far behind. That will create a cleaner commit history, which I want to preserve in the case of the PandaCSS migration so that we can trace regressions more granularly.

Reply:
Sounds good!